recipes/abseil/all/test_package/conanfile.py

```diff
@@ -7,6 +7,8 @@ class TestPackageConan(ConanFile):
 
     def build(self):
         cmake = CMake(self)
+        if not self.settings.compiler.cppstd:
+            cmake.definitions["CMAKE_CXX_STANDARD"] = 11
```

Comment:
```suggestion
```

Reply:
Yes, it's broken if compiling with standard c++17.